data_synchronizer:
    ros__parameters:
        verbose: false                                         # printing info about syncs if true
        rgb_topic: /camera/color/image_raw                    # input topic with rgb image
        depth_topic: /camera/aligned_depth_to_color/image_raw # input topic with aligned to color depth image
        output_topic: /yolo/input                             # output topic for communication with yolo node
        rgb_vis_topic: /data_synchronizer/rgb                 # topic with rgb from data_synchronizer
        realsense: true                                       # sync odom if false (only gazebo) and use imu if true (only realsense)

fake_realsense:
    ros__parameters:
        verbose: false                                        # printing info about syncs if true
        camera_info_topic: /camera/color/camera_info          # output topic with fake camera_info
        rgb_topic: /camera/color/image_raw                    # output topic with fake rgb image
        depth_topic: /camera/aligned_depth_to_color/image_raw # output topic with fake depth image
        imu_topic: /camera/imu                                # output topic eith fake imu data
        framerate: 15                                         # rgb && depth framerate
        imu_rate: 200                                         # imu publish rate

yolo_dispatcher:
    ros__parameters:
        verbose: false                                        # printing info about time if true
        camera_info_input_topic: /camera/color/camera_info    # input topic with camera_info
        camera_info_topic: /rtabmap/yolo/camera_info          # input topic with camera_info
        yolo_topic: /yolo/output                              # input topic with prediction from yolo
        rgb_topic: /rtabmap/yolo/rgb                          # output topic with rgb image
        depth_topic: /rtabmap/yolo/depth                      # output topic with static_depth image
        mask_topic: /rtabmap/yolo/mask                        # output topic with mask
        odom_topic: /rtabmap/yolo/odom                        # output topic with odom
        detector_topic: /dynamic_detector/input               # output topic with dynamic_depth, mask and bounding boxes
        real_mask_topic: /rtabmap/yolo/real_mask              # output topic with mask of dynamic_obstacles shapes
        realsense: true                                       # true if realsense is in use
